index.js

```diff
@@ -21,6 +23,30 @@ const transformBabel = src => {
   return result;
 };
 
+
+const transformTs = (src, path) => {
```

Comment:
Not sure if introducing a dep on ts-jest for something relatively simple gains us much in this case. For the same reason I agree removing a dep on tsconfig is a good idea, more or less.